proxy/test/CommunityPool.ts

```diff
@@ -251,11 +251,6 @@ describe("CommunityPool", () => {
             communityPoolTester = await deployCommunityPoolTester(contractManager, linkerTester, messageProxyTester);
         });
 
-        it("should be rejected with User should be active", async () => {
```

Comment:
Are non active users allowed to send messages?

Reply:
Yeap - that means that message that they are locked was not transferred in 5 minutes(allowed window to deliver this message)